### DISCUSSION TITLE
What is auto build do ?

### DISCUSSION THREAD

**Discussion by dimaslanjaka** [redhat-developer/vscode-java#3474] [Q&A]
what function of this setting? is i can run `gradlew build` automatically on project changes?

![image](https://github.com/redhat-developer/vscode-java/assets/12471057/e18e2796-322a-4cc0-b942-c520ae8c2116)

**Comment by kenwenzel** [comment]
I think it's the same as in Eclipse IDE. If it is enabled then all Java projects are incrementally compiled after changes of the source code.
This helps to save time when starting the application or tests as everything is already compiled.
